src/components/PasswordToggle/PasswordToggle.tsx

```diff
@@ -49,6 +49,10 @@ export type Props = PropsWithSpread<
      * The content for success validation.
      */
     success?: ReactNode;
+    /**
+     * The content for success validation.
+     */
+    type?: string;
```

Comment:
Would be nice to limit that to just the allowed types ("button", "submit", "reset"):
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type

Reply:
Good idea! done